mffpy/xml_files.py

```diff
@@ -289,6 +309,25 @@ def content(cls, fileDataType: str,  # type: ignore
             }
         }
 
+    def get_content(self):
+        """return info on the associated (data) .bin file"""
+        return {
+            'generalInformation': self.generalInformation,
+            'filters': self.filters,
+            'calibrations': self.calibrations
+        }
+
+    def get_serializable_content(self):
+        """return a serializable object containing
+        info on the associated (data) .bin file"""
+        content = copy.deepcopy(self.get_content())
+        # Convert np.float32 values to float built-in type
+        for value in content['calibrations'].values():
+            channels = value['channels']
+            for channel in channels.keys():
+                channels[channel] = float(channels[channel])
```

Comment:
Why is this necessary? I made a toy example and it looks like np.float32 and floats are generally directly comparable. Do we need 64-bit precision or something? (looks like that's _generally_ the default for floats in python these days https://docs.scipy.org/doc/numpy/user/basics.types.html suggests we need to do this )